src/pages/Home/components/CharacterList.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+export default function CharacterList({list}) {
+
+    return (
+        <div className="CharacterList">
+            {
+                list.map((data) => {
```

Comment:
I felt the character component was not needed since it's just receiving all the data, so in my opinion it looked much cleaner to get rid of the character component, and just do the map here and return the table.